app/shownoter.py

```diff
@@ -59,15 +59,12 @@ def format_links_as_markdown(source):
     links = []
 
     for url in urls:
-        if image_detect(url):
-            link = Image(url)
-        else:
-            link = Link()
-            link.collect_data(url)
+        link = Link()
+        link.collect_data(url)
 
         links.append(link.markdown)
 
-    output = links_to_string(links)
+    output = "\n".join(links)
```

Comment:
I like the join call, however I thought it was going in views.py. As the root functionality is shownoter returning a list object. 

This could be a big change when we start looking at creating an API.